### DISCUSSION TITLE
Manage multiple self-managed/hosted renovate runners

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#23623] [Suggest an Idea]
### Tell us more.

As of know, I have 4 self-managed/hosted renovate runners to support repos on github.com, gitlab.com, self-hosted gitlab instances.
It would be valuable to have one system to manage, configure, monitor the runners fleet.

**Comment by secustor** [comment]
I have started to develop a [Renovate Operator](https://github.com/secustor/renovate-operator), but I have never came around to polish it up as none of my customers had a use case for it. 

You can pick it up, if you want.

**Reply by lmilbaum** [reply]
Thanks @secustor. I'll take a look.
